contracts/wallet/WalletFactory.sol

```diff
@@ -196,7 +136,10 @@ contract WalletFactory is Owned, Managed {
         returns (address _wallet)
     {
         require(_guardian != (address(0)), "WF: guardian cannot be null");
```

Comment:
Maybe we should add a check to make sure the `_owner` is not `address(0)` and that the length of `_modules` is not 0.

Reply:
Since we validate these in the create calls, I didn't think it's necessary but we can indeed do the same validation here via `_validateInputs(_owner, _modules, _label, _guardian);`

Reply:
The problem is that `_validateInputs` also checks that the modules are registered which I feel should not be checked in the ` getAddressForCounterfactualWallet` method.